### DISCUSSION TITLE
Cannot download .graphml and .epgz files

### DISCUSSION THREAD

**Discussion by xsarmalenka** [mkdocs/mkdocs#3323] [Q&A]
I use the same link format for images, .docx, .graphml and .epgz files downloading in my mkdocs. If i run my mkdocs locally everything is OK, but if I publish the same mkdocs on server I cannot download .graph and .epgz files. Mkdocs says 404 on the server but files are in folder, really. What is wrong?

Example of links:
[Parking Guidance System UI - výběr.epgz](media/Parking Guidance System UI - výběr.epgz)
[Rezidenstký modul 2.0 a PGS - výběr.docx](media/Rezidenstký modul 2.0 a PGS - výběr.docx)
[01 Provozní doby - výběr.png](media/01 Provozní doby - výběr.png)
[VizualizaceRezervací.graphml](media/VizualizaceRezervací.graphml)

**Comment by squidfunk** [comment]
The problem is related to the MkDocs live reload server that doesn't know about the mime types of those files. I've encountered the same problem when serving `*.webp` files.
